src/components/Calendar.js

```diff
@@ -0,0 +1,18 @@
+import moment from 'moment';
+import {useState} from 'react';
+import Calendar from 'react-calendar';
+
```

Comment:
if you dont want to use the "button" styling, you can always use the default styling that comes with the calender. 
```
import "react-calendar/dist/Calendar.css";

```